static/js/publisher/form/AccordionHelp.js

```diff
@@ -0,0 +1,58 @@
+import React, { Fragment } from "react";
+import PropTypes from "prop-types";
+
+class AccordionHelp extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.toggleHelp = this.toggleHelp.bind(this);
+
+    this.state = {
+      open: false
+    };
+  }
+
+  toggleHelp() {
+    const { open } = this.state;
+
+    this.setState({
+      open: !open
+    });
+  }
+
+  render() {
+    const { open } = this.state;
+    const { name, children } = this.props;
+
+    const classNames = ["row"];
+    let verb = "Hide";
+    if (!open) {
+      classNames.push("u-hide");
+      verb = "Show";
+    }
+
+    return (
+      <Fragment>
+        <p className={`p-form-help-text ${open ? "u-no-margin--bottom" : ""}`}>
+          <a role="button" onClick={this.toggleHelp}>
+            {verb} {name.toLowerCase()}
+          </a>
+        </p>
+        <div className={classNames.join(" ")}>
+          <div className="col-8">{children}</div>
```

Comment:
If it's expected to have just one column, why using `row` and `col-` classes at all?
But, if we want to reuse it to build something like 2 column markdown help, maybe it can just render anything that it gets in `{children}`? So it will be parent component that will decide how to split content into columns?

```suggestion
          {children}
```